superset-frontend/src/SqlLab/components/ResultSet.tsx

```diff
@@ -481,6 +495,28 @@ export default class ResultSet extends React.PureComponent<
     return <div className="noControls" />;
   }
 
+  renderRowsReturned() {
+    const { results, rows } = this.props.query;
```

Comment:
will `this.props.query` always have a value?

Reply:
yup! It will only render if query.results exists.